setup/operators/installtemplates/rhoda.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: install-dbaas-operator
+objects:
+  - apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: ${DBAAS_NAMESPACE}
+  - apiVersion: operators.coreos.com/v1alpha1
+    kind: CatalogSource
+    metadata:
+      name: dbaas-operator-catalog
+      namespace: ${DBAAS_NAMESPACE}
+    spec:
+      sourceType: grpc
+      image: quay.io/tchughesiv/dbaas-operator-catalog:v0.1
```

Comment:
This is what we use in staging: https://github.com/codeready-toolchain/sandbox-sre/blob/master/resources/sandbox-stage.gb17.p1.openshiftapps.com/install/member/rhoda-operator/dbaas-operator.yaml#L13
Let's double check with the RHODA folks what image we should use.

Reply:
Nvm, I was looking at an older version of the rhoda.yaml. I'll use the latest one.

Reply:
Done in 4ab0c915cf1b96997e79fbe4c034ab5cfa134f00